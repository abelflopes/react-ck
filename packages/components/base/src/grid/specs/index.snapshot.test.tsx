import React from "react";
import { Grid } from "../index";
import renderer from "react-test-renderer";

describe("snapshot GridContainer", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Grid />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});

describe("snapshot GridColumn", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Grid.Column />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
