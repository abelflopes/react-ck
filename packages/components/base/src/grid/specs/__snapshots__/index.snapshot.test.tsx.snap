// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot GridColumn renders correctly 1`] = `
<div
  className="root"
  style={
    {
      "--columns": 0,
      "--index": undefined,
      "--size": undefined,
    }
  }
/>
`;

exports[`snapshot GridContainer renders correctly 1`] = `
<div
  className="root spacing_m align_default wrap"
/>
`;
